### ISSUE TITLE
Console Errors

### ISSUE THREAD

**Issue by x0Z3ro0x** [mcMMO-Dev/mcMMO#4062]
Your commands remain VERY touchy. Sometimes they will work, other times they will just begin spamming errors. It does it with every skill commands such as /mining, /unarmed, etc

I'll keep updating this post with the new errors. This one is for /mcstats.
https://pastebin.com/6pRZaF5q

**Comment by electronicboy** [comment]
You didn't /reload, did you? has your profile data loaded yet? (Check for other errors)

**Comment by x0Z3ro0x** [comment]
Nope, personally idk why that useless server reload command is in, it's only ever caused issues to everyone I know using it.

The errors happen at random, not after a server restart, so I can only assume the data has been loaded. If you educate me on how to do so, I would be happy to tell you for sure if the data has been loaded or not.

No other errors are present, I am a bit of a control freak when it comes to the console print offs. Everything must be in order or I simply don't use it.

**Comment by Quavelen** [comment]
Hey @x0Z3ro0x, please try this a clean install of on the latest builds of Paper and mcMMO.

I am unable to reproduce it, and it does not occur on the official mcMMO minecraft server.

Back when you reported this in 2019, electronicboy's assumption of the reload command was one of the few things that could cause your errors.


Due to inactivity on this issue, I will be closing it. If you would like to keep it open, please reply with the results of the above requested test.

**Comment by x0Z3ro0x** [comment]
Greetings @Quavelen. As you can see, I am not extremely active, hence the 7 day delay. I did want to reply though with updated progress. As of today, 8/6/2020, I can say this issue does not occur anymore. I am running the latest build for paper, and mcMMO. Before replying I had users and myself spam the commands just to double check and the issue is no longer present.

Thank you both for the replies, and I am glad to report it isn't an issue going forward. Hope you both are healthy and have a great day / night!

**Comment by x0Z3ro0x** [comment]
Had another error from a command today. Skill commands remain working just fine without error. However users who used command,

/skillreset <SkillName>

Threw an error. Here it is -

https://pastebin.com/UA62h296

**Comment by Quavelen** [comment]
Just for the sake of fidelity @x0Z3ro0x, would you mind updating Paper to build 245 and seeing it it still occurs?
If it does, then perhaps you have found a bug.

**Comment by x0Z3ro0x** [comment]
245 build and same error.

**Comment by Quavelen** [comment]
Thank you @x0Z3ro0x, I'll give this a poke and then let nossr50 know if I'm able to do it as well.

**Comment by Quavelen** [comment]
Okay, so...

![image](https://user-images.githubusercontent.com/19168008/96951858-f3c3fb00-14dc-11eb-909e-c0f8b37541ac.png)

I just tested this on a local test server with mcMMO 2.1.149 and Paper 245. As you can see in the screenshot, I did not get the skill reset error that you did.

Would you be willing to provide a full log file, @x0Z3ro0x? Preferably one with the startup info where it states what plugins are loading, etc.

**Comment by x0Z3ro0x** [comment]
I tried entering the command as well and did not get the error. So I looked through the logs of the players who reported it. They were using 

/skillreset SkillName

rather than

/skillreset player SkillName

I just managed to reproduce the error using that. I would be happy to provide logs, just let me know if you still need them after reading the above.

**Comment by Quavelen** [comment]
Thank you, @x0Z3ro0x. No need for the logs now.
Using `/skillreset SkillName` without the `PlayerName` argument did indeed cause the error.

![image](https://user-images.githubusercontent.com/19168008/96955869-2d4d3400-14e6-11eb-9d07-b117b5c97b87.png)

![image](https://user-images.githubusercontent.com/19168008/96955862-2a524380-14e6-11eb-9923-c44935d88ec5.png)


I've let @nossr50 know on Discord.
